src/main/java/com/netgrif/application/engine/pdf/generator/service/fieldbuilder/TextFieldBuilder.java

```diff
@@ -104,4 +114,16 @@ private String shortenFileName(String fileName) {
         }
         return fileName;
     }
+
+    private boolean isCurrencyField(LocalisedField field) {
+        return field.getComponent() != null &&
+                Objects.equals(field.getComponent().getName(), "currency") &&
+                field.getComponent().getProperties() != null &&
+                field.getComponent().getProperties().containsKey("code") &&
```

Comment:
are all these properties now required for number fields?
is this a breaking change?
should we provide default values if the properties are missing?

Reply:
These are not required. They are used in case of currency field, they do not have any default value.

Reply:
if the values are not required for currency fields, then if one of them is absent, the method `isCurrencyField` will return `false` even if it is a currency field. Is this intended, or am I misunderstanding something?

Reply:
if the values are not required for currency fields, then if one of them is absent, the method `isCurrencyField` will return `false` even if it is a currency field. Is this intended, or am I misunderstanding something?

Reply:
Yes, you are right, in that case, in PDF there will be only a number, without currency symbol.